<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSvgEditorClass</class>
 <widget class="QMainWindow" name="KSvgEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KSvgEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="KToolBar" name="toolBar" native="true">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KSvgMainWindow" name="editPage" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KCanvasParamsBar" name="paramBar" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(255, 85, 127);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KToolBar</class>
   <extends>QWidget</extends>
   <header location="global">ktoolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KSvgMainWindow</class>
   <extends>QWidget</extends>
   <header location="global">ksvgmainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KCanvasParamsBar</class>
   <extends>QWidget</extends>
   <header location="global">kcanvasparamsbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ksvgeditor.qrc"/>
 </resources>
 <connections/>
</ui>
